stats/lib/platform-stats-fetchers.js

```diff
@@ -1,16 +1,16 @@
 import assert from 'http-assert'
 import { today, yesterday } from './request-helpers.js'
 
-/** @typedef {import('@filecoin-station/spark-stats-db').Queryable} Queryable */
+/** @typedef {import('./typings.js').DateRangeFilter} DateRangeFilter */
 
 const ONE_DAY = 24 * 60 * 60 * 1000
 
 /**
- * @param {Queryable} pgPool
+ * @param {Object} pg 
```

Comment:
```suggestion
 * @param {Queryable} pg 
```

I believe this still follows the `Queryable` interface. `Object` is too unspecific.